src/datagrid/column-service/column-service.jsx

```diff
@@ -202,6 +203,63 @@ export default {
         break;
       }
 
+      case 'multiselect': {
+        const selectOptions = col.selectComponentOptions ||
+          props.selectComponentOptions.get(column.columnKey);
+        const selectTranslations = col.selectComponentTranslations ||
+          {
+            placeholder: props.intl.formatMessage({ id: 'Grid.FloatingSelect.Select' }),
+            noResultsText: props.intl.formatMessage({ id: 'Grid.FloatingSelect.NoResults' }),
+            selected: '{n} selected',
```

Comment:
Is there a reason this is not translated? What about the commented line below (on line 214)

Reply:
There is no reason. I just forgot to normalize this after testing it.